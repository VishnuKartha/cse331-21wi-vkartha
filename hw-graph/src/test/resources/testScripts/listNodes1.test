# Creates a single graph and adds one node to it and lists it.

# Create a Graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Lists the nodes
ListNodes graph1