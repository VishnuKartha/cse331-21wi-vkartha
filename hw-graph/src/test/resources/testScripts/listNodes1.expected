# Creates a single graph and adds one node to it and lists it.

# Create a graph
created graph graph1

# Add a Node
added node n1 to graph1

# Lists the nodes
graph1 contains: n1